### ISSUE TITLE
Connect My Custom Domain with Gophish URL 

### ISSUE THREAD

**Issue by ajbmntro** [gophish/gophish#2944]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: v0.12.1

Brief description of the issue: i want to connect my custom url to url campaign in gophish, because it should 127.0.0.1:80 not my custom domain. how to connect my custom domain to my gophish url

What are you expecting to see happen? : when i blast the email, and they click my link they can load my page with internet. 

What are you seeing happen?: dont know how to connect my custom domain to gophish url.

**Comment by glennzw** [comment]
Hi there, thanks for reaching out. You'll need to host gophish on a public IP address, and point your (sub)domain to that IP. e.g:

Run gophish on public host 12.13.14.15
Login to your domain provider and set an A record my.phishdomain.com to 12.13.14.15

I use DigitalOcean for my gophish hosting (you can use my referral [here](https://m.do.co/c/92d344b69f50) to get $200 credit).

I hope that helps!

**Comment by ajbmntro** [comment]
how to run gophish on public host?

is it like this? 

<img width="957" alt="image" src="https://github.com/gophish/gophish/assets/92260208/5f27e8e9-b66c-4c82-9052-5adcfe1f8df1">

**Comment by glennzw** [comment]
Yes that looks right. Now you have a public IP (43.250.77.68). You'd want to point your domain to that IP.

**Comment by wand3rlust** [comment]
> how to run gophish on public host?
> 
> is it like this?
> <img alt="image" width="957" src="https://user-images.githubusercontent.com/92260208/261993148-5f27e8e9-b66c-4c82-9052-5adcfe1f8df1.png">

Exposing Admin portal over Internet could potentially ruin the campaign. Running it on localhost and forwarding the port over SSH is a better alternative.

**Comment by Hosted-EU** [comment]
There’s A New Phishing Tool Out There.
It Can Generate URLs For Phishing Pages.
And Able To Steal Cookies from Phishing Pages To Bypass 2FA 🔥

Watch Video On How It Works : https://youtube.com/watch?v=BJv3jFuJJOw
